app/components/EventItem/index.tsx

```diff
@@ -185,7 +185,7 @@ const EventItem = ({
                     <Image
                       src={event.cover}
                       placeholder={event.coverPlaceholder}
-                      alt={`Event cover image - ${event.title}`}
+                      alt={`Forsidebilde til ${event.title}`}
```

Comment:
I think it should be "Forsidebildet til" instead of "Forsidebilde til"

Reply:
I agree - changed it now 😄

Reply:
Me neither 💀